spillr_paper.Rmd

```diff
@@ -190,23 +201,22 @@ means <- combo |>
 
 zeros_means <- left_join(zeros, means, by = "correction")
 levels(zeros_means$correction)[2] <- "\\texttt{spillR}"
-levels(zeros_means$correction)[3] <- "\\texttt{spillR} (fast)"
+levels(zeros_means$correction)[3] <- "\\texttt{spillR} (naive)"
 levels(zeros_means$correction)[4] <- "\\texttt{CATALYST}"
 
-zeros_means |>
-  kableExtra::kbl(
-    caption = "Additional summaries of the data underlying Figure \\ref{fig:method-example}C.", 
-    booktabs = TRUE, digits = 2, escape = FALSE
-    )
+# zeros_means |>
+#   kableExtra::kbl(
+#     caption = "Additional summaries of the data underlying Figure \\ref{fig:method-example}C.", 
+#     booktabs = TRUE, digits = 2, escape = FALSE
+#     )
 ```
 
-Panel C displays the distribution of our target marker, CD3 (Yb173Di), before and after spillover correction. Our compensation method, `spillR`, masks most markers in the low counts up to two, \textcolor{red}{between 50\% to 100\% in the medium counts range between two and four}, and keeps counts above four. Larger counts are not affected by the correction, as can be seen by the overlap of both curves. In contrast, `CATALYST` shifts large counts to the left, and shifts medium counts to low counts or zero counts. Zero counts (or equivalently `NA` counts for `spillR`) and mean counts are shown in Table \@ref(tab:method-example-summary). 
-\textcolor{red}{
-As we do not know the true mean in this case, the comparison of means here serves merely as an illustration of the main differences between \texttt{spillR} and \texttt{CATALYST} and not to identify the correct method. Our baseline method \texttt{spillR} (fast) is similar to \texttt{CATALYST} in low and medium range, but keeps higher counts unchanged.
-}
-
+Panel C displays the distribution of our target marker, CD3 (Yb173Di) before and after spillover correction. \textcolor{red}{We observe few real counts (red) below a value of $2$, so although all methods perform strong compensation in this range, there is little visible change. From $2$ onward there is a clear distinction between the methods. `CATALYST`, like our baseline `spillR (naive)`, compensates nearly all counts below the second peak of the raw counts (red) as spillover. By contrast, `spillR` compensates only where the relative frequency of the raw counts (red) match the density of spillovermarkers in the bead experiment shown in panel A, and does not remove a part of the first peak as a result. While `CATALYST` shifts the distribution of large counts (around 6) slightly to the left, our methods leave them unaffected as the bead experiment shows no spillover in this region.}
```

Comment:
Panel C displays the distribution of the target marker, CD3 (Yb173Di), before and after spillover correction. Our compensation method, `spillR`, masks most markers in the low counts up to two, \textcolor{red}{between 50\% to 100\% in the medium counts range between two and four}, and keeps counts above four unchanged. Larger counts are not affected by the correction, as can be seen by the overlap of both curves.
Likewise, our baseline method \texttt{spillR} (naive) strongly compensates the low to medium range counts (between ...) that align with spillover observed in the beads experiments, but, in contrast to \texttt{CATALYST}, leaves higher counts where there was no spillover observed in the beads experiment unchanged.
In contrast to spillR, `CATALYST` shrinks all counts, that is, also large counts are shifted to the left, and medium counts are shifted to low counts or zero counts (`spillR` provides an option to choose between setting spillover counts to `NA` or zero).